packages/web3-eth-accounts/src/account.ts

```diff
@@ -1,22 +1,173 @@
 import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { keccak256 } from 'ethereum-cryptography/keccak';
+import {
+	TransactionFactory,
+	FeeMarketEIP1559TxData,
+	AccessListEIP2930TxData,
+	TxData,
+} from '@ethereumjs/tx';
+import { ecdsaSign, ecdsaRecover } from 'secp256k1';
 import {
 	toChecksumAddress,
 	bytesToHex,
 	sha3Raw,
 	HexString,
 	isBuffer,
 	isValidString,
+	Address,
+	isHexStrict,
+	utf8ToHex,
+	hexToBytes,
 } from 'web3-utils';
-import { InvalidPrivateKeyError, PrivateKeyLengthError } from 'web3-common';
+import {
+	InvalidPrivateKeyError,
+	PrivateKeyLengthError,
+	UndefinedRAWTransactionError,
+	SignerError,
+	InvalidSignatureError,
+} from 'web3-common';
+import {
+	signatureObject,
+	signFunction,
+	signResult,
+	signTransactionFunction,
+	signTransactionResult,
+} from './types';
 
 // TODO Will be added later
 export const encrypt = (): boolean => true;
 
-// TODO Will be added later
-export const sign = (): boolean => true;
+/**
+ * Hashes the given message. The data will be UTF-8 HEX decoded and enveloped as follows: "\x19Ethereum Signed Message:\n" + message.length + message and hashed using keccak256.
+ */
 
-// TODO Will be added later
-export const signTransaction = (): boolean => true;
+export const hashMessage = (message: string): string => {
+	const messageHex = isHexStrict(message) ? message : utf8ToHex(message);
```

Comment:
We should use the [validateStringInput](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-utils/src/validation.ts#L135) method before this for `message`

Reply:
isHexString() will check if its type is string, else inside utf8ToHex() validateStringInput is already checked. so no need to do it explicitly here,

Reply:
`isHexString()` will check if its type is `string`, else inside `utf8ToHex()` validateStringInput is already checked. so no need to do it explicitly here